### PR TITLE
Person scopes

### PR DESCRIPTION
**Pull request by jensljungblad**
- [x] Add new pending status on match
  - [x] Add `started_at` to match
  - [x] Create `pending` scope on match
  - [x] Make it possible to go from pending to active to inactive
  - [x] Filter out unmatched from interested & engaged
- [x] Add new scopes to person
  - [x] Pending matches
  - [x] Active matches
  - [x] Inactive matches
- [x] Name all the things and put correct labels on scopes

### CODE REVIEW

**Comment by jensljungblad** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;8VAnng9D&#x2F;72-nar-jag-listar-personer-sa-vill-jag-ha-flikar-for-intresserade-intervjuade-erbjudna-matchning-matchade-och-avslutade-personer"> När jag listar personer, så vill jag ha flikar för intresserade, intervjuade, erbjudna matchning, matchade och avslutade personer</a>

File: app/models/person.rb [hunk 1]
```diff
@@ -10,8 +10,23 @@ class Person < ActiveRecord::Base
...
+           ON matches.newcomer_id = people.id
+           OR matches.established_id = people.id").where("matches.id IS NULL")
+  }
+
+  scope :matched, lambda {
+    joins("INNER JOIN matches
+           ON matches.newcomer_id = people.id
+           OR matches.established_id = people.id")
+  }
+
```
Comment (juliaramstedt) [inline app/models/person.rb]: Yay joins!

**Comment by juliaramstedt** [comment]
👍

**Comment by juliaramstedt** [comment]
The joins are pretty confusing to me, but they look smart. What about tests? ;-)

**Comment by jensljungblad** [comment]
What about them?

File: app/models/person.rb (hunk 1 above)
Comment (jensljungblad) [inline app/models/person.rb]: Det här är förmodligen inte så bra, men det verkar funka! Får se hur det funkar när det finns massa data...

**Pull request merged**